images/build/setcap/README.md

```diff
@@ -1,19 +1,24 @@
-setcap
+### setcap
+
 This image is based on debian-base and installs the libcap2-bin package. The
 main use of this image is to apply `CAP_NET_BIND_SERVICE` to the kube-apiserver
 binary so that it can a bind to ports less than 1024 and still be run as non
 root.
 
 This image is compiled for multiple architectures.
 
-How to release
+### How to release
+
 If you're editing the Dockerfile or some other thing, please bump the TAG in the Makefile.
 
+```console
 Build and  push images for all the architectures
```

Comment:
Would you mind moving this out of code block or making it a comment?

Reply:
Done! Thanks.